.github/workflows/test.yml

```diff
@@ -174,3 +174,52 @@ jobs:
           cp functionalAccounts.json.test functionalAccounts.json
           docker-compose up --build -d
           npm run test:api
+
+  generate_and_test_openapi_client:
+    name: Generate an test OpenAPI Client libraries
+    needs: [install-and-cache]
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checkout your code
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      # Generate your OpenAPI document (if you don't write it manually)
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Restore node_modules
+        id: cached-node-modules
+        uses: actions/cache@v3
+        with:
+          path: node_modules
+          key: node-modules-${{ hashFiles('package-lock.json') }}-${{ env.NODE_VERSION }}
+
+      # Todo: May need reworking in order to not block
+      - name: Start Backend for later tests and generate openapi.json
```

Comment:
Please use the  same flow as in the API tests (l.132ff)  ( you need to start a mongodb instance and use the correct configs)